WooCommerce/WooCommerceTests/ViewModels/Receipts/ReceiptEligibilityUseCaseTests.swift

```diff
@@ -205,92 +210,133 @@ final class ReceiptEligibilityUseCaseTests: XCTestCase {
 
         // When
         let isEligible: Bool = waitFor { promise in
-            sut.isEligibleForFailedPaymentEmailReceipts(paymentGatewayID: CardPresentPaymentsPlugin.wcPay.gatewayID, onCompletion: { result in
+            sut.isEligibleForFailedPaymentEmailReceipts(paymentGatewayID: "woocommerce-stripe", onCompletion: { result in
                 promise(result)
             })
         }
 
         // Then
+        XCTAssertFalse(isEligible)
+    }
+
+    func test_meetsOrderStatusRequirement_with_completed_status_returns_true() {
```

Comment:
nit: would be nice keeping the Given/When/Then comments for consistency and clearer structure of the test case. Same for other `meetsOrderStatusRequirement` test cases as well.

Reply:
True that! https://github.com/woocommerce/woocommerce-ios/pull/15558/commits/fcbbcb0ca4804755f8decdff84272a370890ac49